deepchecks/nlp/utils/text_properties.py

```diff
@@ -282,7 +282,7 @@ def _predict(text, classifier, kind):
             return v['score']
         elif kind == 'fluency':
             label_value = 'LABEL_1'
-        elif kind == 'fluency':
+        elif kind == 'formality':
```

Comment:
I know this is not related to this PR but I regret this code and think the treatment of prediction for each property should be within each property, as in this implementation the logic is split between two functions and it's confusing...

Reply:
i can change it none the less, any objections @yromanyshyn ?